libtermbench/termbench.cpp

```diff
@@ -103,263 +101,274 @@ void Benchmark::runAll()
         buffer->clear();
     }
 }
-
+//
 void Benchmark::summarize(std::ostream& os)
 {
-    os << fmt::format("All {} tests finished.\n", results_.size());
-    os << fmt::format("---------------------\n\n");
+    os << std::format("All {} tests finished.\n", results_.size());
+    os << std::format("---------------------\n\n");
     auto const gridCellCount = width_ * height_;
 
-    std::chrono::milliseconds totalTime{};
+    std::chrono::milliseconds totalTime {};
     size_t totalBytes = 0;
     for (auto const& result: results_)
     {
         Test const& test = result.test.get();
         auto const bps = double(result.bytesWritten) / (double(result.time.count()) / 1000.0);
         totalBytes += result.bytesWritten;
         totalTime += result.time;
-        os << fmt::format(
-            "{:>20}: {:>3}.{:04} seconds, {}/s (normalized: {}/s)\n",
-            test.name,
-            result.time.count() / 1000,
-            result.time.count() % 1000,
-            sizeStr(bps),
-            sizeStr(bps / static_cast<double>(gridCellCount))
-        );
+        os << std::format("{:>40}: {:>3}.{:04} seconds, {}/s (normalized: {}/s)\n",
+                          test.name,
+                          result.time.count() / 1000,
+                          result.time.count() % 1000,
+                          sizeStr(bps),
+                          sizeStr(bps / static_cast<double>(gridCellCount)));
     }
 
     auto const bps = double(totalBytes) / (double(totalTime.count()) / 1000.0);
-    os << fmt::format(
-        "{:>20}: {:>3}.{:04} seconds, {}/s (normalized: {}/s)\n",
-        "all tests",
-        totalTime.count() / 1000,
-        totalTime.count() % 1000,
-        sizeStr(bps),
-        sizeStr(bps / static_cast<double>(gridCellCount))
-    );
+    os << std::format("{:>40}: {:>3}.{:04} seconds, {}/s (normalized: {}/s)\n",
+                      "all tests",
+                      totalTime.count() / 1000,
+                      totalTime.count() % 1000,
+                      sizeStr(bps),
+                      sizeStr(bps / static_cast<double>(gridCellCount)));
     os << "\n";
-    os << fmt::format(" screen size: {}x{}\n", width_, height_);
-    os << fmt::format("   data size: {}\n", sizeStr(static_cast<double>(testSizeMB_ * 1024 * 1024)));
+    os << std::format(" screen size: {}x{}\n", width_, height_);
```

Comment:
while I am not against it, would that still work with older Ubuntu versions? 🤔 

I think mixing fmt::format along with std::format use will at least not hurt the final binary. hmmm....

Reply:
We do not use fmt::format here anymore